firmware/controllers/trigger/trigger_central.cpp

```diff
@@ -311,7 +311,7 @@ void hwHandleVvtCamSignal(TriggerValue front, efitick_t nowNt, int index) {
 			front == TriggerValue::RISE ? SHAFT_PRIMARY_RISING : SHAFT_PRIMARY_FALLING, nowNt);
 		// yes we log data from all VVT channels into same fields for now
 		tc->triggerState.vvtSyncGapRatio = vvtDecoder.triggerSyncGapRatio;
-		tc->triggerState.vvttoothDurations0 = vvtDecoder.toothDurations[0];
+		tc->triggerState.vvttoothDurations0 = (float)NT2US(vvtDecoder.toothDurations[0]);
```

Comment:
why do we have to cast to float? would this value ever have numeric in it?

i.e. would it be better to just change data type to int32?

Reply:
I don't think we need sub-microsecond precision, so I'll make it uint32_t...